client/signup/main.jsx

```diff
@@ -332,6 +333,7 @@ class Signup extends React.Component {
 			isNewUser,
 			isNewSite,
 			hasCartItems,
+			isNewishUserSite,
```

Comment:
Can you think of a better name for this? `isNewishUserSite` is what I came up with before, but it's not very clear. The `-ish` is actually not clear at all. What about `isNew7DUserSite` or something more descriptive could work also. I think we could error on the side of this variable being more descriptive.